packages/bruno-app/src/components/ResponsePane/QueryResult/index.js

```diff
@@ -18,15 +18,22 @@ const formatResponse = (data, mode, filter) => {
     return '';
   }
 
-  if (mode.includes('json')) {
+  let isTrueJSON;
+  try {
+    isTrueJSON = typeof JSON.parse(JSON.stringify(data)) === 'object';
+  } catch (error) {
+    console.log('Error parsing JSON: ', error.message);
```

Comment:
Console output is not required here, I think?
```suggestion
    console.log('Error parsing JSON: ', error.message);
```

Reply:
Sure, that is not required. It was for inspection purposes. We can remove that.